### PR TITLE
Managing response error codes

### PR DESCRIPTION
**Pull request by franramirez688**
Added a simple error codes manager

Detecting different errors and launching specific exceptions.

### CODE REVIEW

File: bwproject.py [hunk 1]
```diff
@@ -231,15 +243,11 @@ def bare_request(verb, address_root, address_suffix, access_token="",
...
+            raise exc.BrandwatchApiException(str(e))
         else:
             errors = response_json.get('errors')
             if errors:
-                logger.error("There was an error with this "
-                             "request: \n{}\n{}\n{}".format(response.url, data,
-                                                            errors))
-                raise BrandwatchApiException(errors)
-
+                _raise_bw_exception(response, data, errors[0])
```
Comment (champ) [inline bwproject.py]: We better iterate on the errors since it’s a list.

**Review by champ — CHANGES_REQUESTED** [review]
Looks good. Just needs to address the multiple errors case.

File: bwproject.py (hunk 1 above)
Comment (franramirez688) [reply bwproject.py]: I can only raise one exception :D but I can change exception format if there are several errors

**Review by champ — APPROVED** [review]
Looks good.

**Pull request merged**